app/styles/modules/_global_header.scss

```diff
@@ -10,16 +10,43 @@ header {
 
   h1 {
     @include vertical-align;
-    color: #ffffff;
     font-size: 24px;
+    left: 44px;
     margin: 0;
+
+    a {
+      color: #ffffff;
+      text-decoration: none;
+    }
   }
 
-  #user {
-    display: none;
+  #user-info {
+    margin-top: -18px;
+    position: absolute;
+    top: 50%;
+
+    img {
+      border-radius: 50%;
+      height: 36px;
+      width: 36px;
+    }
+
+    nav {
+      background: #fff;
+      border: 1px solid #eee;
```

Comment:
We should use `#ffffff` and `#eeeeee` for consistency (although I think @johngruen may eventually want to refactor these out into variables or something so we can track color use a bit easier).